src/amd_layout.js

```diff
@@ -1,9 +1,16 @@
 (function(global, factory) {
-  if (typeof define === 'function' && define.amd)
+  if (typeof module === "object" && typeof module.exports === "object" )
+    module.exports = global.document
+      ? factory(global, true)
+      : function(w) {
+        if (!w.document) throw new Error( "Zepto requires a window with a document" )
+        return factory(w)
+      }
+  else if (typeof define === 'function' && define.amd)
     define(function() { return factory(global) })
   else
     factory(global)
-}(this, function(window) {
+}(typeof window !== "undefined" ? window : this, function(window) {
```

Comment:
When I using webpack. I found `__webpack_require__` execute the module closure in this way:

```javascript
// ...
var module = installedModules[moduleId] = {
    exports: {},
    id: moduleId,
    loaded: false,
    hot: hotCreateModule(moduleId),
    parents: hotCurrentParents,
    children: []
}
modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId))
// ...
```

It cause `this` in zepto is equal to module.exports (exports: {}), so the `window` will be `{}`.
That why cause `Uncaught TypeError: Cannot read property 'createElement' of undefined`

So we need to use `window` variable first, I think this is for nodejs environment, to expose factory function.